hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/ObjectStore.java

```diff
@@ -178,6 +178,18 @@ public S3SecretValue getS3Secret(String kerberosID, boolean createIfNotExist)
     return proxy.getS3Secret(kerberosID, createIfNotExist);
   }
 
+  /**
+   * Set secretKey for accessId.
+   * @param accessId
+   * @param secretKey
+   * @return S3SecretValue <accessId, secretKey> pair
+   * @throws IOException
+   */
+  public S3SecretValue setSecret(String accessId, String secretKey)
```

Comment:
I named this `setSecret` instead of `setS3Secret` because it is more generic. Or maybe I should just use `setS3Secret` to keep the naming consistent with the rest?

Reply:
perhaps setS3Secret is more appropriate.

Reply:
Good idea. Done